src/azure-cli/azure/cli/command_modules/storage/commands.py

```diff
@@ -277,11 +294,6 @@ def get_custom_sdk(custom_module, client_factory, resource_type=ResourceType.DAT
                                        validator=process_blob_download_batch_parameters)
         g.storage_custom_command_oauth('delete-batch', 'storage_blob_delete_batch',
                                        validator=process_blob_delete_batch_parameters)
-        g.storage_custom_command_oauth('show', 'show_blob', table_transformer=transform_blob_output,
-                                       client_factory=page_blob_service_factory,
-                                       doc_string_source='blob#PageBlobService.get_blob_properties',
-                                       exception_handler=show_exception_handler)
```

Comment:
Is the `blob show` command of Track 1 completely off-line? All the Azure Stacks will be running Track 2 in the future?

Reply:
what do you mean for off-line? cut off?
track 1 command is still in use for Azure Stack. and track 2 will change to new implementation.